### DISCUSSION TITLE
Problems with set_active

### DISCUSSION THREAD

**Discussion by ferm10n** [smartin015/continuousprint#134] [Q&A] [answered]
I tried following one of the examples:
https://github.com/smartin015/continuousprint/blob/3cc83069d11789e9aeef71aaefb567fb86205d33/api_examples/example.py#L11-L16

I'd get a 200 response, but no change and no logs.

It wasn't until I sent the string `'true'` that something worked. This was just surprising to me, given what was in the docs.

This worked:
```python
requests.post(octoprintOrigin + '/plugin/continuousprint/set_active', data={ 'active': 'true' }, headers={"X-Api-Key": apiKey})
```

But this didn't:
```python
requests.post(octoprintOrigin + '/plugin/continuousprint/set_active', data={ 'active': True }, headers={"X-Api-Key": apiKey})
```

I just wanted to understand if I'm missing something / document working around this issue.

Thanks!

**Comment by smartin015** [comment, marked as answer]
Huh no, that definitely sounds like a bug to me. I converted this to an issue (see #135) and will investigate a bit.
